test/aries-js-worker/test/verifiable/verifiable.js

```diff
@@ -133,6 +133,25 @@ async function verifiableStore(newAries, mode = wasmMode) {
         )
     })
 
+    it(modePrefix + "Alice makes sure that the DID is resolvable", async function () {
+        let success = false
+        for (var i = 0; i < 10; i++) {
+            await axios.get(`${environment.HTTP_LOCAL_RESOLVER_URL}/` + did.id)
+                .then(function(response) {
+                    success = true
+                })
+                .catch(function(error) {
+                    console.log('will try to resolve again : attempt=',i+1);
+                });
+
+            if (success) {
+                break
+            }
+
+            await new Promise(r => setTimeout(r, 2000));
```

Comment:
can we sleep for less time? Our build's duration has gotten longer and longer...

Reply:
we should be looping around smaller timeouts.